### ISSUE TITLE
Chained comparisons cause incorrect warnings

### ISSUE THREAD

**Issue by nedbat** [edx/edx-lint#43]
This line of code:
```
assertTrue(0 <= npoints <= 1)
```
causes:
```
[C7690(wrong-assert-type), CustomResponseTest.test_function_code_single_input] assertTrue(0 <= npoints <= 1) should be assertLessEqual
```
